### ISSUE TITLE
How do you write underline text?

### ISSUE THREAD

**Issue by Xeoncross** [fogleman/gg#82]
I would like to write underline text but I'm not sure if that is something I need to define with an external package like github.com/golang/freetype or something I have to manually calculate and DrawLine() for.

**Comment by nigeltao** [comment]
> I don't think the underlying freetype package supports underlining or strikethrough.

Correct, the underlying freetype package does not support underlining or strikethrough.

As for `golang.org/x/image/font/sfnt`, the intention was to replace `github.com/golang/freetype`, including rasterizing glyphs, but it is sadly unfinished. As of a few years ago, I'm no longer paid to work on it, and I haven't had the spare time to finish it.
